src/Controllers/DevCheckController.php

```diff
@@ -43,8 +43,11 @@ public function index($request)
             $suite = $name;
         }
 
-        $checker = new EnvironmentChecker($suite, 'Environment status');
-        $checker->setRequest($request);
+        /** @var EnvironmentChecker */
+        $checker = EnvironmentChecker::create($suite, 'Environment status')
+            ->setRequest($request)
+            ->setIncludeDetails(false);
+
         $checker->init($this->config()->permission);
```

Comment:
May as well string this onto the end with the other method calls since you've changed the calls above to be fluent

Reply:
`init` is not fluent though

Reply:
If it's at the end of the chain that's fine - I'll leave this up to you though, I don't have strong feelings either way.